amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/InitializeTask.java

```diff
@@ -101,12 +101,17 @@ public TaskResult call() {
             final long startTime = System.currentTimeMillis();
             try {
                 shardRecordProcessor.initialize(initializationInput);
+                initializeSuccess = true;
                 log.debug("Record processor initialize() completed.");
             } catch (Exception e) {
                 applicationException = true;
                 throw e;
             } finally {
-                MetricsUtil.addLatency(scope, RECORD_PROCESSOR_INITIALIZE_METRIC, startTime, MetricsLevel.SUMMARY);
+                MetricsUtil.addSuccessAndLatency(scope,
+                        RECORD_PROCESSOR_INITIALIZE_METRIC,
+                        initializeSuccess,
+                        startTime,
+                        MetricsLevel.SUMMARY);
```

Comment:
We dont have this in public KCL version?

Reply:
This doesn't show up in the public KCL verison. According to the [docs](https://docs.aws.amazon.com/streams/latest/dev/kinesis-dg.pdf), there's a Success metric, but it's not being properly emitted here

I can update the code to 
```java
MetricsUtil.addSuccess(scope, null, initializeSuccess, MetricsLevel.SUMMARY);
```

in order to follow the current docs